### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by eyalbe4**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.

---
This PR includes multiple improvements to the README documentation. See the new README in my forked branch.
It includes:
1. Phrasing and grammar corrections.
2. Layout and styling improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,42 +11,48 @@
...
 #### GitHub
-For security reasons, Frogbot is not triggered automatically.
-After you create a new pull request, the maintainer of the git repository can trigger Frogbot to scan the pull request from the pull request UI. The scan output will include only new vulnerabilities added by the pull request.
-Vulnerabilities that aren't new, and existed in the code prior to the pull request creation, will not be included in the
+After you create a new pull request, the maintainer of the Git repository can trigger Frogbot to scan the pull request from the pull request UI. 
+
+> **_NOTE:_** The scan output will include only new vulnerabilities added by the pull request.
+Vulnerabilities that aren't new, and existed in the code before the pull request was created, will not be included in the
 report. In order to include all of the vulnerabilities in the report, including older ones that weren't added by this
 PR, use the JF_INCLUDE_ALL_VULNERABILITIES environment variable.
```
Comment (omerzi) [inline README.md:51]: Should we add a note that everyone will see these vulnerabilities and this env var is not recommended for general use?

File: README.md [hunk 2]
```diff
@@ -55,31 +61,38 @@ PR, use the JF_INCLUDE_ALL_VULNERABILITIES environment variable.
...
 #### GitLab
 
-For security reasons, Frogbot is not triggered automatically.
+For security reasons, Frogbot cannot be automatically triggered.
 After you create a new pull request, the maintainer of the git repository can trigger Frogbot to scan the pull request
 from the pull request UI. The scan output will include only new vulnerabilities added by the pull request.
-Vulnerabilities that aren't new, and existed in the code prior to the pull request creation, will not be included in the
+Vulnerabilities that existed in the code prior to the pull request creation, will not be included in the
 report. In order to include all of the vulnerabilities in the report, including older ones that weren't added by this
 PR, use the JF_INCLUDE_ALL_VULNERABILITIES environment variable.
```
Comment (omerzi) [inline README.md:69]: Shouldn't we add this inside a NOTE as well?

**Review by omerzi — CHANGES_REQUESTED** [review]
LGTM overall :) Added some comments

File: README.md (hunk 1 above)
Comment (eyalbe4) [reply README.md:51]: I think the current statement should be sufficient. People will probably understand the consequences.

File: README.md (hunk 2 above)
Comment (eyalbe4) [reply README.md]: You're right. Done.

**Pull request merged**